Taskfile.yml

```diff
@@ -1226,6 +1229,14 @@ tasks:
     cmds:
       - 'if [ -z "{{.DOCKER_REGISTRY}}" ]; then echo "Error: DOCKER_REGISTRY must be set"; exit 1; fi'
       - az acr login --name {{.DOCKER_REGISTRY}}
+    vars:
+      DOCKER_REGISTRY:
+        sh: |
+          if [ ! -z "${DOCKER_REGISTRY}" ]; then
+            echo "${DOCKER_REGISTRY}"
+          else
+            echo "{{.DOCKER_REGISTRY}}"
+          fi
```

Comment:
I don't think you need to do that. Task [will look in the environment](https://taskfile.dev/usage/#variables) for `.DOCKER_REGISTRY` if it's not already in scope.

Reply:
Tested without this and confirmed it works.